web_src/css/base.css

```diff
@@ -1337,6 +1340,10 @@ overflow-menu .overflow-menu-items .item {
   margin-bottom: 0 !important; /* reset fomantic's margin, because the active menu has special bottom border */
 }
 
+overflow-menu .ui.label {
+  margin-left: 7px !important;
+}
+
```

Comment:
This rule was previously in repo only, but we can make this apply to all such menus.

Reply:
Could there be a comment to explain why "ui label" needs such "margin-left" ?

Reply:
It's a space-saving measure. Default is `1em` from `.ui.menu .item > .label:not(.floating)`.

Reply:
Added comment in https://github.com/go-gitea/gitea/pull/29982/commits/fa5167ec4f2c139a25c738defd15e90853a7148e.